projects/simple-nix/default.nix

```
@@ -3,4 +3,21 @@
 with nixpkgs.pkgs;
 with gitignore;
 
-haskellPackages.callCabal2nix "{{name}}" (gitignoreSource ./.) {}
+let
+  hdl = "verilog";
```

```suggestion
  hdl = "verilog";
  topModule = "Example.Project";
```

Pull out the module name into a separate variable, to make it easier to change.